connectivity-graph/connectivity_graph.py

```diff
@@ -28,6 +28,8 @@
 
 #===============================================================================
 
+from all_graphs import graph_from_knowledge
```

Comment:
this will probably cause a circular import here, if it is moved to the end it does not (bad, but will get the code to run)